content/en/docs/instrumentation/java/manual.md

```diff
@@ -35,6 +37,86 @@ the most important piece of telemetry source-identifying info.
 
 For example:
 
+<details>
+  <summary>Maven dependencies. Click me.</summary>
+  
+  ```
+  
+  <project>
+      <dependencyManagement>
+          <dependencies>
+              <dependency>
+                  <groupId>io.opentelemetry</groupId>
+                  <artifactId>opentelemetry-bom</artifactId>
+                  <version>{{% param javaVersion %}}</version>
+                  <type>pom</type>
+                  <scope>import</scope>
+              </dependency>
+          </dependencies>
+      </dependencyManagement>
+      
+      <dependencies>
+          <dependency>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-api</artifactId>
+          </dependency>
+          <dependency>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-sdk</artifactId>
+          </dependency>
+          <dependency>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-exporter-otlp</artifactId>
+          </dependency>
+          <dependency>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-semconv</artifactId>
+              <version>{{% param javaVersionSemConv %}}</version>
```

Comment:
WDYT about using the `javaVersion` param for now?
```suggestion
                  <version>{{% param javaVersion %}}-alpha</version>
```
It'll continue to be alpha for a long while anyway.